ingesters/winevents/main.go

```diff
@@ -189,22 +188,26 @@ type levelLogger struct {
 }
 
 // levelLogger implements the log.LevelRelay interface
-func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg []byte) error {
+func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg, rawMsg string) error {
 	switch lvl {
 	case log.DEBUG:
 		if debugOn {
-			fmt.Fprintln(os.Stdout, string(msg))
+			fmt.Fprintln(os.Stdout, rawMsg)
```

Comment:
Do we ever want log messages on stdout? Even debug should go to stderr right?

Reply:
no, debug is only done when you are actively debugging with a specific flag.  ITs not part of the traditional log chain